### PR TITLE
Navbar + Footer fixes

### LINKED ISSUE #1032: Navbar + Footer fixes
## Navbar
- BH logo blurry on Chrome/Safari
- BH logo strange height
- MLH logo SVG needs to be flush w/ top of screen

## Footer
Looks a bit strange

Screen Shot 2020-12-11 at 2 06 29 PM

### PR DESCRIPTION
**Pull request by ww2497**
resolves #1032 

image
image

### CODE REVIEW

**Comment by ww2497** [comment]
Unsure about what the BH logo stuff mentioned in the issue is referring to

**Comment by peterkos** [comment]
I'll give a review in the morning!

File: sass/main.scss [hunk 1]
```diff
@@ -637,20 +636,28 @@ footer {
...
+        
+        p {
+            font-size: 0.95em;
         }
 
         // FA brand icons
         i {
             font-size: 3em;
-            padding-right: 10px;
+            padding: 0 5px;
```
Comment (peterkos) [inline sass/main.scss:660]: Good catch on the padding here! Although style-wise, lets add units to the `0` to keep it inline with the `px` on the right:
```suggestion
            padding: 0px 5px;
```
(rest of stylesheet follows this convention, aside when unit is undefined, like for `marign: 0 auto;`)

File: index.html [hunk 2]
```diff
@@ -362,24 +362,35 @@ <h1>Leadership Team</h1>
...
-            <a href="https://github.com/codeRIT" target="_blank">
-                <i class="fab fa-github"></i>
-            </a>
-            <br />
-            <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">code of conduct | </a>
-            <a href="https://www.rit.edu/academicaffairs/policiesmanual/c070" target="_blank">privacy policy</a>
-            <p>copyright &#169; 2020 codeRIT</p>
+            <div>
+                <h2>contact us</h2>
+                <p>
```
[the remark this answers is not in the document]
Comment (peterkos) [reply index.html:367]: (for all lines in this PR where applicable)

File: sass/main.scss [hunk 3]
```diff
@@ -637,20 +636,28 @@ footer {
     overflow-x: clip;
 
     #footer-content {
+        display: flex;
```
Comment (peterkos) [inline sass/main.scss:639]: I like this solution 😄

**Review by peterkos — CHANGES_REQUESTED** [review]
Great work! Couple small changes.

File: sass/main.scss [hunk 4]
```diff
@@ -637,20 +636,28 @@ footer {
...
         text-align: center;
 
-        p, a, i {
-            margin-bottom: 10px;
+        h2 {
+            margin-bottom: -5px;
+        }
+        
+        p {
+            font-size: 0.95em;
```
[the remark this answers is not in the document]
Comment (ww2497) [reply sass/main.scss]: 0.95 matches the design more closely with the 'contact us' and 'hello@coderit.org' text having similar widths, but I suppose this is fine

**Review by peterkos — APPROVED** [review]
LGTM :shipit:

**Pull request merged**